databroker/tests/test_broker.py

```diff
@@ -197,6 +197,14 @@ def test_uid_lookup():
     assert_raises(ValueError, lambda: db[uid[0]])
 
 
+def test_find():
+    insert_run_start(time=100., scan_id=1,
+                     owner='nedbrainard', beamline_id='example',
+                     uid=str(uuid.uuid4()))
+    result = db(start_time=0, stop_time=ttime.time())
```

Comment:
a range of 99 - 101 and checking that this runstart is in there would probably be better.  

This will find _everything_ we inserted and only checks that we found _anything_.